src/management/Akka.Management/Routes/AddressRouteProvider.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System.Threading.Tasks;
+using Akka.Actor;
+using Akka.Http.Dsl;
+using Akka.Management.Dsl;
+using Akka.Remote;
+using Ceen;
+using Newtonsoft.Json;
+using Route = System.ValueTuple<string, Akka.Http.Dsl.HttpModuleBase>;
+
+namespace Akka.Management.Routes;
+
+public class AddressRouteProvider: IManagementRouteProvider
+{
+    public Route[] Routes(ManagementRouteProviderSettings settings)
+    {
+        return new HttpAddressRoutes().Routes;
+    }
+}
+
+internal class HttpAddressRoutes : HttpModuleBase
+{
+    public Route[] Routes => new Route[] { ("/remote/address", this) };
+    
+    public override async Task<bool> HandleAsync(IAkkaHttpContext context)
+    {
+        if (context.HttpContext.Request.Method.ToLowerInvariant() != "get")
+            return false;
+        
+        // Check that remoting is in effect
+        if (((ExtendedActorSystem)context.ActorSystem).Provider is not IRemoteActorRefProvider actorProvider)
```

Comment:
lol, I get why this is necessary but it would be pretty funny if someone managed to get Akka.Management up and running without clustering or remoting.